modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,227 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-sno-on-ibm-z-lpar_{context}"]
+= Installing {sno} in an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+.Prerequisites
+
+* If you are deploying a single-node cluster there are zero compute nodes; the Ingress Controller pods run on the control plane nodes. In single-node cluster deployments, you must configure your application ingress load balancer to route HTTP and HTTPS traffic to the control plane nodes. See the _Load balancing requirements for user-provisioned infrastructure_ section for more information.
+
+.Procedure
+
+. Set the {product-title} version by running the following command:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version> <1>
+----
++
+<1> Replace `<ocp_version>` with the current version, for example, `latest-{product-version}`
```

Comment:
Needs punctuation:

<1> Replace `<ocp_version>` with the current version. For example, use `latest-{product-version}`.

Reply:
These modules have the same sentence I'll add them to the PR
modules/install-sno-ibm-z-kvm.adoc
modules/install-sno-ibm-z.adoc